src/notifications/index.ts

```diff
@@ -0,0 +1,33 @@
+const nodemailer = require('nodemailer');
+
+let transporter = nodemailer.createTransport({
+    service: 'Gmail',
+    auth: {
+        user: 'meowl.notifications@gmail.com',
+        pass: 'Thisisapassword'

```

Comment:
Use an environment variable for password and change password

Reply:
Yes this is a big oversight on my part. Thanks for pointing it out.